docs/README.md

```diff
@@ -0,0 +1,88 @@
+# Current outline
+This is subject to change.
+* Introduction
+* How to play Hypermine
+    * Controls
+    * Config file
+    * The save file (with a warning that compatibility between versions of Hypermine are not guaranteed and another warning to back up save files)
+    * Setting up multiplayer
+* Background math
+    * Linear algebra
+        * (This can link to external resources, but readers should be guided on what parts of linear algebra are worth learning, and making these docs self-contained would be a good long-term goal. If we do use external links, we should include a date so that readers know when health of each link was last checked.)
+        * Vectors
+        * Matrices, matrix-vector multiplication, and its meaning
+        * Matrix-matrix multiplication and its meaning
+            * (We should likely explain both the "transformation" and "change of basis" interpretations)
+        * 3D examples (assuming previous sections have used 2D examples)
+        * Dot products
+        * Projections, reflections, rotations
+        * Homogeneous coordinates and translations
+    * Spherical geometry
+        * (We want such a section because it's a good segue to problem solving techniques for hyperbolic geometry problems)
+        * Representing points as unit vectors
+        * Projections, reflections, rotations, translations (which are rotations expressed differently)
+    * Hyperbolic geometry
+        * Minkowski space (with the "inner product" as the main difference)
+        * Representing points as "normalized" vectors
+        * Projections, reflections, rotations, translations, horo-rotations
+        * A note about floating point precision
+        * (Do we put advanced shape-casting math here? Probably not.)
+* Tiling the world
```

Comment:
I propose leaning heavily on the analogous 2D tiling for ease of visualization wlog, once the analogy has been established.